tutorials/export/exporting_for_web.rst

```diff
@@ -133,12 +133,15 @@ engine is started from within a valid input event handler. This requires
 Audio
 ~~~~~
 
-Chrome restricts how websites may play audio. It may be necessary for the
-player to click or tap or press a key to enable audio.
+Some browsers restrict autoplay for audio on websites. The easiest way around this limitation is to request the
+player to click, tap or press a key/button to enable audio, for instance when displaying a splash screen at the start of your game.
 
 .. seealso:: Google offers additional information about their `Web Audio autoplay
              policies <https://sites.google.com/a/chromium.org/dev/audio-video/autoplay>`__.
 
+            Apple's Safari team also posted additional information about their `Auto-Play Policy Changes for macOS 
```

Comment:
```suggestion
             Apple's Safari team also posted additional information about their `Auto-Play Policy Changes for macOS 
```
I think this not lining up with the word "policies" on the earlier line is causing the build error, but I'm not 100% sure, still should be fixed though.